easybuild/easyblocks/m/molpro.py

```diff
@@ -106,6 +106,13 @@ def configure_step(self):
                     self.cfg.update('configopts', "-mpp -mppbase %s" % os.environ['MPI_INC_DIR'])
                 else:
                     raise EasyBuildError("$MPI_INC_DIR not defined")
+                if self.toolchain.mpi_family() == toolchain.OPENMPI:
+                    self.cfg.update('configopts', "-openmpi")
+                elif self.toolchain.mpi_family() == toolchain.INTELMPI:
+                    self.cfg.update('configopts', "-intel-mpi")
```

Comment:
This break the existing `Molpro-2012.1.27-intel-2015a.eb` easyconfig:

```
unknown Molpro option: -intel-mpi
```

Reply:
Wheren't you going to deprecate intel <= 2016a?? :-)

Reply:
@boegel Repeating: Do you have 2012.1.27 available? I only have 2012.1.52
I.e., can i have a copy of that source to look at.